From f25eed0008813e488e1acd5a49adccd959237c6d Mon Sep 17 00:00:00 2001
From: Marc Fournier <marc.fournier@camptocamp.com>
Date: Thu, 5 Nov 2015 10:23:19 +0100
Subject: [PATCH 07/14] varnish: remove unused variable

This was used in a DEBUG statement I didn't check in.

(cherry picked from commit 08bd4dd86e0fcb6828819cdf6bb3ae2115b1b8f4)
---
 src/varnish.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/varnish.c b/src/varnish.c
index cd52217..682e362 100644
--- a/src/varnish.c
+++ b/src/varnish.c
@@ -141,7 +141,6 @@ static int varnish_monitor (void *priv, const struct VSC_point * const pt) /* {{
 	uint64_t val;
 	const user_config_t *conf;
 	const char *class;
-	const char *ident;
 	const char *name;
 
 	if (pt == NULL)
@@ -151,7 +150,6 @@ static int varnish_monitor (void *priv, const struct VSC_point * const pt) /* {{
 
 #if HAVE_VARNISH_V4
 	class = pt->section->fantom->type;
-	ident = pt->section->fantom->ident;
 	name  = pt->desc->name;
 
 	if (strcmp(class, "MAIN") != 0)
@@ -159,7 +157,6 @@ static int varnish_monitor (void *priv, const struct VSC_point * const pt) /* {{
 
 #elif HAVE_VARNISH_V3
 	class = pt->class;
-	ident = pt->ident;
 	name  = pt->name;
 
 	if (strcmp(class, "") != 0)
-- 
2.5.0

